curvefs/src/client/warmup/warmup_manager.h

```diff
@@ -224,9 +236,9 @@ class WarmupManager {
      * @return true
      * @return false warmupProcess has been added
      */
-    virtual bool AddWarmupProcess(fuse_ino_t key) {
+    virtual bool AddWarmupProcess(fuse_ino_t key, WarmupStorageType type) {
         WriteLockGuard lock(inode2ProgressMutex_);
-        auto ret = inode2Progress_.emplace(key, WarmupProgress());
+        auto ret = inode2Progress_.emplace(key, WarmupProgress(0, 0, type));
```

Comment:
add another constructor for `WarmupProgress` that only accept type ?

Reply:
> add another constructor for `WarmupProgress` that only accept type ?

fix